repo/packages/P/postgresql-admin/0/marathon.json.mustache

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{postgresql-admin.cpus}},
+    "mem": {{postgresql-admin.mem}},
+    "instances": 1,
+    "env": {
+        "DB_HOST": "{{networking.postgres_host}}.marathon.l4lb.thisdcos.directory",
+        "DB_PORT": "{{networking.postgres_host_port}}"
+    },
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.postgresql-admin-docker}}",
+            "forcePullImage": false,
+            "network": "BRIDGE",
+            "portMappings": [
+            {
+                "containerPort": 80,
+                "hostPort": 0,
+                "servicePort": {{networking.port_external}}
```

Comment:
made optional and default to false.

I can see use cases in which Dan won't let people use Admin Router, and will just provision apps through the external node.